ddl/resourcegrouptest/resource_group_test.go

```diff
@@ -119,16 +125,20 @@ func TestResourceGroupBasic(t *testing.T) {
 	}
 	g = testResourceGroupNameFromIS(t, tk.Session(), "y")
 	checkFunc(g)
-	tk.MustExec("alter resource group y BURSTABLE RU_PER_SEC=5000")
+	tk.MustExec("alter resource group y BURSTABLE RU_PER_SEC=5000 QUERY_LIMIT=(EXEC_ELAPSED_IN_SEC='15s' ACTION KILL)")
```

Comment:
It occurs to me that as we can pass any duration to `EXEC_ELAPSED_IN_SEC`, does `IN_SEC` make any sense?

Reply:
I agree. `EXEC_ELAPSED` is better. @songrijie